wolfssl/wolfcrypt/port/xilinx/xil-versal-trng.h

```diff
@@ -0,0 +1,46 @@
+/* xil-versal-trng.h
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+
+#ifndef WOLF_XIL_CRYPT_VERSAL_TRNG_H
+#define WOLF_XIL_CRYPT_VERSAL_TRNG_H
+
+#ifdef versal
+#include "xparameters.h"
+#include "xil_types.h"
+#include "xsecure_mailbox.h"
+#include "xsecure_defs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+int wc_versal_trng_init(byte* nonce, word32 nonceSz);
```

Comment:
Prefer camel case for new functions added in. (exception is if in one of our older files, like ecc, then it would match the pattern existing in the file).

Reply:
Done